v22.1/restore.md

```diff
@@ -344,7 +373,7 @@ By default, tables and views are restored to the database they originally belong
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> RESTORE bank.customers FROM '2021/03/23-213101.37' IN 's3://{bucket_name}/{path/to/backup}?AWS_ACCESS_KEY_ID={key_id}&AWS_SECRET_ACCESS_KEY={access_key}'
```

Comment:
tough that you have to copy and paste all this stuff for each external storage format! I wonder if it's even neccesary to have examples for each format? perhaps a seperate issue

Reply:
Yes... too true. I have a docs issue to rework some markdown liquid template magic post release work. Apologies for the triple review.